internal/operators/lvm/lvm_operator.go

```diff
@@ -145,7 +145,9 @@ func (o *operator) GetHostRequirements(ctx context.Context, cluster *common.Clus
 	}
 
 	role := common.GetEffectiveRole(host)
-	if role == models.HostRoleMaster && !swag.BoolValue(cluster.SchedulableMasters) {
+	areSchedulable := common.AreMastersSchedulable(cluster)
```

Comment:
can we cover this case with a unit-test?

Reply:
ok, maybe I'm missing something (because the PR is missing some details)
what's the expected outcome and what's the actual outcome that this PR handles?

Reply:
this value `cluster.SchedulableMasters` is a toggle / optional config - by default is set to false. On Full cluster, Masters that are not Schedulabe do not require additonal resources.

the bug is that on SNO / Compact `cluster.SchedulableMasters=false` and return empty struct instead of the required values.

Reply:
K I've improved the test to better reflect the use cases.
+ I've tested with and without the fix - now it will fail when there is incorrect value

Reply:
This is awesome, thank you!